app/models/concerns/rubygem_searchable.rb

```diff
@@ -49,16 +49,35 @@ def as_indexed_json(_options = {}) # rubocop:disable Metrics/MethodLength
     settings number_of_shards: 1,
              number_of_replicas: 1,
              analysis: {
-               analyzer: {
-                 rubygem: {
+               filter: {
+                 autocomplete_ngram: {
```

Comment:
Can you please share your findings related to ngram being faster than suggest API?
What was your method of testing this and how much miliseconds faster was ngram on your local setup?

Reply:
As I ran the test, I think the execution time is so small and it could be negligible. In addition, actually completion suggester seems faster. See my gist how I've collect a result.
https://gist.github.com/shunshunNi/81604414181a5ee69ae3f5f497791b2b